docs/carma_installation.md

```diff
@@ -57,7 +57,10 @@ containing a base-line configuration of this system are available upon request.
 `./carma_build –a ~/carma_ws/autoware.ai`
 9-1- In case of missing dependencies  install with the command:
 `sudo apt-get install ros-kinetic-XXX`
+apt-get install libfftw3-3
+apt-get install libfftw3-dev
+sudo apt-get install libproj-dev
```

Comment:
Move these three lines to section "6-1-1- Get Autoware prerequisites" 
Also add the -y option